fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/digitalcash/DigitalCashSendFragment.java

```diff
@@ -145,6 +145,7 @@ private void setUpTheAdapter() throws KeyException {
     List<String> myArray;
     try {
       myArray = digitalCashViewModel.getAttendeesFromTheRollCallList();
+      assert myArray != null;
```

Comment:
I don't think this can be null. In case no RC is found an exception is thrown I believe

Reply:
Yes indeed, however the IDE was not that happy about it so I just added it to remove the warnings